examples/secp256k1/README.md

```diff
@@ -39,19 +43,41 @@ Assuming you've your local environment set up, let's go to the spore secp256k1 e
 cd examples/secp256k1
 ```
 
-The example code is stored in the [apis directory](./apis), feel free to review the code or change the data of spore/cluster you want to create on-chain, or even treat the examples as your experimental fields.
+The example code is stored in the [examples/secp256k1/apis](./apis) directory. Feel free to review and change anything you want, treating the examples as your experimental playground or a code sandbox to explore and play around with.
 
 To run an example, for instance if you want to create a spore on-chain, you can run the [apis/createSpore.ts](./apis/createSpore.ts) example in your terminal:
 
 ```shell
 ts-node apis/createSpore.ts
 ```
 
-The example should construct and send a transaction that creates a spore, and after sending the transaction, it should return a `Transaction Hash` for you to view the details of the transaction on [CKB Explorer](https://pudge.explorer.nervos.org/).  
+The example should construct and send a transaction that creates a spore on-chain. After sending the transaction, it should return a `Transaction Hash` for you to review the details of the transaction on [CKB Explorer](https://pudge.explorer.nervos.org/).
```

Comment:
This example constructs and sends a transaction that creates a spore on-chain. Once the transaction is sent, a `Transaction Hash` should be returned so that you can review the transaction details on [CKB Explorer].